apps/search-widget-demo/src/App.svelte

```diff
@@ -1,13 +1,18 @@
 <script lang="ts">
-  import { setLang } from 'libs/search-widget/src/core/i18n';
+  import { setLang } from '../../../libs/search-widget/src/core/i18n';
   import { onMount } from 'svelte';
   import { NucliaWidget } from '../../../libs/search-widget/src';
   import { NucliaSearchBar, NucliaSearchResults } from '../../../libs/search-widget/src/_video-widget';
 
-  let selected = 'two-widgets';
-  let widget: NucliaWidget;
+  let selected = 'input';
+  console.log(NucliaWidget, NucliaSearchBar, NucliaSearchResults);
```

Comment:
do we want to keep this console.log?

Reply:
yes, we need to do something with these classes because if we don't, the compiler will ignore the import and it won't work (the compiler just thinks `<nuclia-search>` is a component that might or might not exist, it does not understand it is actually provided by the `NucliaSearch` class)